src/run.f90

```diff
@@ -7547,7 +7521,7 @@ subroutine WriteDailyResults(DAP, WPi)
         end if
 
         ! 5. Relative cover of weeds
-        if (GetCCiActual() <= 0.0000001_dp) then
+        if (GetCCiActual() <= 0.000001_dp) then
```

Comment:
idem, and perhaps we'd need a full code search on these types of hardwired thresholds

Reply:
done